__tests__/endpoint.test.js

```diff
@@ -78,4 +71,32 @@ describe('Endpoint Tests', () => {
               });
           });
     })
+    describe('/api/articles/', () => {
+        test.only('200: response with an articles array of article object, which should contain all collumns except body, and be in data descending order', () => {
```

Comment:
To ensure we aren't breaking other tests, be sure to remove test.only when pushing up (and give all tests a run to be on the safe side!)